### DISCUSSION TITLE
UI & UX ideas for AFK

### DISCUSSION THREAD

**Discussion by MSghais** [AFK-AlignedFamKernel/afk_monorepo#24] [Ideas]
GM!

We create this discussion to share and debate about the UI/UX of AFK.

You can share your feedback, suggestions, and ideas with a Figma if possible.

We can reward you on Only Dust for a good Figma UI/UX board about the modifications, changes, and improvements you want to make in the AFK app!

**Comment by 0xZyrick** [comment]
Hi team,

I've uploaded some wireframe screens focusing on the navigation UX. 
Future updates will include annotations for clarity. 
Let me know your thoughts or feedback!

https://www.figma.com/design/yN681uZ3P0FLARfvKR0JcI/AFK-PROJECT?m=auto&t=dXjRbhO4cyGm7PtB-6

**Reply by 0xZyrick** [reply]
Gm Gm 
Desktop screen has been uploaded in the figma file above

**Comment by 0xZyrick** [comment]
Gm gm 
I notice a bug on the "pump" screen updated earlier so I took it out . I will be re-updating back
